K branch3

wonderEx24 opened the pull request:
resources.loadをしたいのですが、pで撮影してからなんらかのキーでロードをし、ゲーム内で読み込めるようにしたいですが、どうすればいいかがよくわからないです。

wonderEx24 committed:
2025 9/14

当たり判定とへずもく　異界50階の再クリア簡単すぎで草　もはや風雲50階より弱いのほんまに草

wonderEx24 committed:
2025 9/21

現状確認とスクリーンショット機能　無量空所（漢字わからん）定期

chosakuken commented:
# 2025/09/28用 @ぶり
ねこけんにやってほしいことの流れを纏めておくね！
## 流れ
1. `Asset/Resources` フォルダを作る
2. スクショの画像ファイルの行先を `Asset/Resources` にする
3. ボタンを押したら、その画像を `Resources.LoadAll` 関数で読み込む

この一連の流れは、このサイトも参考になるよ！ [公式ドキュメント](https://docs.unity3d.com/ja/2018.4/ScriptReference/Resources.html)、[参考](https://www.sejuku.net/blog/55260)
とはいえ、使っている関数が違うので、ぶりが作ったプロトタイプをぜひ参考にしてほしい！！

## Resources フォルダとは？？
フォルダの中身を一発で取得できるわよ。たとえ、動的になっていても問題なく取得できると思う！基本的な使い方は、`Resources.Load` なんだけど、`Resources.LoadAll` を使ってほしい！
## 最後に
一応Windows環境で、`Resources.Load` で読み込んで表示するプログラムは作ってみました！参考にしてね
[ぶりのプロジェクト](https://github.com/chosakuken/share_unity/tree/instance_of_resources)

wonderEx24 committed:
2025 9/28

スクリーンショット読み込みと90分格闘したけど結果的に無理だった。どこぞの霊長類最強と90分格闘するよりマシなはずなのに...

wonderEx24 committed:
add gitignore

wonderEx24 committed:
add gitignore

gitignoreの追加

wonderEx24 committed:
Remove ignore file

wonderEx24 committed:
10/5

いろいろコードいじったせいで70分くらいの努力が無に返って萎えた。

wonderEx24 committed:
10/12

今日は寝不足と疲労と怪我で死にそうだった。だいぶしんどかったけどなんとか画像の読み込みが完成した。
月̶曜̶日̶祝̶日̶な̶の̶に̶バ̶イ̶ト̶入̶っ̶て̶る̶の̶ほ̶ん̶ま̶に̶お̶も̶ん̶な̶い̶。̶
なんなら火曜から金曜までテストなのにまだ勉強できてないし、課題なんも終わってないし、ほんとにどうすればいいんですか？（※金曜、土曜はバイト）

wonderEx24 committed:
ファイル整理、DC Code消した

サモランやりすぎてスプラのプレイ時間が1週間で25時間ぐらい増えてた。後悔なんてあるわけない。

wonderEx24 committed:
再度DC Code破壊

最高神の力で削除しました。
